core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -847,6 +847,33 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     }
   }
 
+  it should "delete an Azure workspace" in withTestDataServices { services =>
+    val workspaceName = s"rawls-test-workspace-${UUID.randomUUID().toString}"
+    val workspaceRequest = MultiCloudWorkspaceRequest(
+      testData.testProject1Name.value, workspaceName, Map.empty, WorkspaceCloudPlatform.Azure, "fake_region"
+    )
+    when(services.workspaceManagerDAO.getWorkspace(any[UUID], any[OAuth2BearerToken])).thenReturn(
+      new WorkspaceDescription().azureContext(new AzureContext()
+        .tenantId("fake_tenant_id")
+        .subscriptionId("fake_sub_id")
+        .resourceGroupId("fake_mrg_id")
+      )
+    )
+
+    val workspace = Await.result(services.mcWorkspaceService.createMultiCloudWorkspace(workspaceRequest), Duration.Inf)
+
+    val deleteWorkspace = Await.result(
```

Comment:
⛏️ : this variable name could be more descriptive, something like `deletedBucketName` perhaps?